.github/workflows/build.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: gradle/wrapper-validation-action@v1
-      - uses: actions/setup-java@v3
+      - uses: actions/setup-java@v3.6.0
```

Comment:
`v3` equates to the latest version of major version 3, or 3.6.0, automatically.

Reply:
did not know that, nifty